.github/workflows/deploy.yml

```diff
@@ -26,6 +26,9 @@ jobs:
         env:
           NEXT_PUBLIC_AI_AGENT_URL: ${{ vars.NEXT_PUBLIC_AI_AGENT_URL }}
 
+      - name: Generate Sitemap
```

Comment:
Just remove this step, since the generate sitemap script (`bun .github/scripts/generate-sitemap.js`) will have already run when executing `npm run build`.